Formula/osm2pgsql.rb

```diff
@@ -24,7 +24,7 @@ def install
     # This is essentially a CMake disrespects superenv problem
     # rather than an upstream issue to handle.
     lua_version = Formula["lua"].version.to_s.match(/\d\.\d/)
-    inreplace "cmake/FindLua.cmake", "LUA_VERSIONS5 5.3 5.2 5.1 5.0",
+    inreplace "cmake/FindLua.cmake", "LUA_VERSIONS5 5.4 5.3 5.2 5.1 5.0",
```

Comment:
Can't we just replace the whole line?

Reply:
@SMillerDev can you explain what you mean exactly? Line 55 of the file `cmake/FindLua.cmake` is:
```
    set(LUA_VERSIONS5 5.4 5.3 5.2 5.1 5.0)
```
so do you mean:
```
    inreplace "cmake/FindLua.cmake", "set(LUA_VERSIONS5 5.4 5.3 5.2 5.1 5.0)",
                                     "set(LUA_VERSIONS5 #{lua_version})"
```

Reply:
I was hoping that we could just replace the entire line, independent of the specific Lua versions on it.

Reply:
So, perhaps something like:
```
    inreplace "cmake/FindLua.cmake", "set(LUA_VERSIONS5 /(\d\.\d ?)/)",
                                     "set(LUA_VERSIONS5 #{lua_version})"
```

Reply:
Sounds better to me

Reply:
Done, thanks for the suggestion.